tests/scripts/all.sh

```diff
@@ -1892,6 +1892,16 @@ skip_suites_without_constant_flow () {
     export SKIP_TEST_SUITES
 }
 
+skip_all_except_given_suite () {
```

Comment:
Because one is grepping inside the files, and one is grepping in filenames, I think the result of merging them would end up more complicated than simply having two functions. So unless the second reviewer disagrees, I would prefer to keep them separate.

Reply:
Agreed